### PR TITLE
LWDEV-10486 Map walletId to Client id in trade volume service

### PR DESCRIPTION
**Pull request by olpapchenko**

### CODE REVIEW

File: src/main/kotlin/com/lykke/trade/volume/monitoring/service/config/ClientAccountssConfig.kt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package com.lykke.trade.volume.monitoring.service.config
+
+class ClientAccountsConfig(
```
Comment (barsukovskii) [inline src/main/kotlin/com/lykke/trade/volume/monitoring/service/config/ClientAccountssConfig.kt]: wrong file name

File: src/main/kotlin/com/lykke/trade/volume/monitoring/service/process/impl/TradeVolumesProcessorImpl.kt [hunk 2]
```diff
@@ -52,15 +56,18 @@ class TradeVolumesProcessorImpl(private val targetAssetId: String,
         }
     }
 
-    private fun processTradeVolume(eventSequenceNumber: Long, tradeVolume: TradeVolume): TradeVolumePersistenceData {
+    private fun processTradeVolume(eventSequenceNumber: Long, tradeVolume: TradeVolume): TradeVolumePersistenceData? {
+        val clientId = clientAccountsCache.getClientByWalletId(tradeVolume.walletId)
+        if (clientId == null) {
+            LOGGER.warn(eventSequenceNumber, "Can not find client by wallet: ${tradeVolume.walletId}")
```
Comment (barsukovskii) [inline src/main/kotlin/com/lykke/trade/volume/monitoring/service/process/impl/TradeVolumesProcessorImpl.kt]: Propose to throw an exception here -the method `process` will handle it and send notification to slack

Comment (olpapchenko) [reply src/main/kotlin/com/lykke/trade/volume/monitoring/service/process/impl/TradeVolumesProcessorImpl.kt]: fixed

File: src/main/kotlin/com/lykke/trade/volume/monitoring/service/config/ClientAccountssConfig.kt (hunk 1 above)
Comment (olpapchenko) [reply src/main/kotlin/com/lykke/trade/volume/monitoring/service/config/ClientAccountssConfig.kt]: fixed

**Review by barsukovskii — APPROVED** [review]

**Pull request merged**